opentelemetry-etw-metrics/src/exporter/mod.rs

```diff
@@ -39,265 +41,135 @@ impl Debug for MetricsExporter {
     }
 }
 
+fn emit_export_metric_service_request(
+    export_metric_service_request: &ExportMetricsServiceRequest,
+) -> MetricResult<()> {
+    let mut encoding_buffer = Vec::with_capacity(export_metric_service_request.encoded_len());
```

Comment:
Not a blocker for this PR but we should be able to reuse the `vec` instead of allocating a new one for each export. `MetricExporter` struct can hold a `vec` as one of its fields and keeping using that `vec` for each export.

Reply:
I make a tweak that's part of the way towards this improvement. Now `emit_export_metric_service_request` takes a `&mut Vec<u8>` as a parameter to avoid allocating a fresh `Vec<u8>` every invocation. Now a new vector is only initialized once per `export`.